### ISSUE TITLE
docs/index.MD

### ISSUE THREAD

**Issue by kmadathil** [kmadathil/sanskrit_parser#105]
@avinashvarna 
I noticed that the index.md symlink you placed was breaking github's pages build. I have git rmed that symlink

Your intention was to ensure that https://kmadathil.github.io/sanskrit_parser/ doesn't return a 404 right?

**Comment by kmadathil** [comment]
I suspect changing the link to ./build/html/index.html will work. Will that suffice? 
(I think the issue was that github pages gets only the docs subdirectory, and hence can't have symlinks to our root)

**Comment by avinashvarna** [comment]
Is the build fixed? Looks like the 404 still exists though.

**Comment by kmadathil** [comment]
Yes, the build is clean now. You have to go to  https://kmadathil.github.io/sanskrit_parser/docs/build/html , though
